src/includes/go.inc.js

```diff
@@ -360,6 +360,10 @@ function drupalgap_back() {
       });
     }
     else if (active_page_id == '_drupalgap_splash') { return; }
+    else if (drupalgap.back_path == ''){
```

Comment:
@signalpoint, thanks for the feedback, been a little swamped with site launches lately. For now I overwrote the drupalgap_back in my theme.js file and everything is working as expected. I'll get back into this when I can get a chance to address those points.